packages/gestalt/src/Button.js

```diff
@@ -1,6 +1,6 @@
 // @flow strict
 
-import React, { useRef, type Element } from 'react';
+import * as React from 'react';
```

Comment:
@AlbertCarreras curious, what was the reason for this change?

Reply:
I was trying to stick to one pattern in particular. TBDiscussed